app/app.go

```diff
@@ -687,9 +655,8 @@ func NewRollapp(
 		evm.NewAppModule(app.EvmKeeper, app.AccountKeeper, app.GetSubspace(evmtypes.ModuleName)),
 		feemarket.NewAppModule(app.FeeMarketKeeper, app.GetSubspace(feemarkettypes.ModuleName)),
 		// Evmos app modules
-		transferModule,
+		transfer.NewAppModule(app.TransferKeeper),
```

Comment:
will this refactor not effect initialize stack and expected state for each step?